pages/api/graphql-rest.page.ts

```diff
@@ -110,6 +118,25 @@ class MpdxRestApi extends RESTDataSource {
 
     return getTaskAnalytics(data);
   }
+
+  async getFourteenMonthReport(
```

Comment:
@TheNoodleMoose Did you have any thoughts about how to get these out of the main file? I didn't explore it at all but this file will probably keep growing. I keep meaning to look into https://www.graphql-modules.com/ but haven't. Idk if it's compatible with `apollo-datasource-rest` which is what we haven't split up yet.

Reply:
Yeah, I tried out some solution where I extracted the function to a separate file, and passed the ```this.get``` as an argument. If I remember it worked but didn't feel like the best solution. The issue with that solution is that the file is still growing, just a little slower haha. I think looking into graphql modules would be the next step to help improve this. A few articles I looked at recommended it, but of course, they had a completely different setup and were not using ```apollo-data-source-rest``` so I'm not sure of the compatibility.

Reply:
Ya we should look into that a little more. Not urgent though.